doc/tests/manual/protocol.md

```diff
@@ -1,359 +1,104 @@
 # Manual Test Protocol
 
-## TC-001 - Login
-
-- Description: Successfully login to PermaplanT
-- Preconditions:
-- Test Steps:
-  1. Press the login button on the navbar.
-  2. Enter credentials.
-  3. Press Login.
-- Expected Result:
-  - Get redirected to the homepage.
-  - Be logged in.
-  - See a Hello message pop up.
-- Actual Result:
-- Test Result:
-- Notes:
-
-## TC-002 - Translation
-
-- Description: Switch language on website
-- Preconditions:
-- Test Steps:
-  1. Change the language in the navbar
-- Expected Result:
-  - Language should be changed after selecting a new one.
-- Actual Result:
-- Test Result:
-- Notes:
-
-## TC-003 - Use Case: Map Creation
-
-- Description: Successfully create a new map.
-- Preconditions:
-  - Be logged in.
-- Test Steps:
-  1. Go to the map editor page.
-  2. Create a new map.
-  3. Enter valid stuff.
-- Expected Result:
-  - Map should be successfully created.
-- Actual Result:
-- Test Result:
-- Notes:
-
-## TC-004 - Map Creation (Negative)
-
-- Description: Can't create a map with a name that exists
-- Preconditions:
-  - Be logged in.
-- Test Steps:
-  1. Go to the map editor page.
-  2. Create a map with a name that already exists.
-  3. Enter valid stuff.
-- Expected Result:
-  - Map should not be created.
-  - Error message pops up on the top right.
-- Actual Result:
-- Test Result:
-- Notes:
-
-## TC-005 - Plant Search
+## TC-001 - Plant Search
```

Comment:
I think we can remove these numbers, which benefit do they provide?

Reply:
They are a good reference and also when you have multiple tests like `Map Editor Guided Tour` you dont need to name them all differently.

Reply:
It was just my comments having a number, not the feature files.

Reply:
Yeah, but implicitly there can be several scenarios per feature file. Maybe we should simply have several test protocol files.

Is there a specific reason why we don't go the full way and write a `.feature` file with comments for additional information for manual testing?

Reply:
We could try it, its just not common practice to use feature files like this I think.

Reply:
We can also simply split up and keep the syntax as proposed here?